### ISSUE TITLE
Adjusted Prices to Include Dividend Adjustment

### ISSUE THREAD

**Issue by ryantjo** [polygon-io/issues#83]
It appears the adjusted prices only account for splits, for fully adjusted prices dividends should also be accounted for. 

It would be useful to be able to request unadjusted, split-adjusted, split & dividend-adjusted prices

**Comment by jrbell19** [comment]
It is planned, however I do not have a concrete timeframe to expect the dividend adjustments. Any news on the progress of this will be will be posted here as well as our [blog. ](polygon.io/blog)

**Comment by pav-user** [comment]
The lack of properly adjusted prices makes technical analysis much more complicated. Any updates or an ETA for fully adjusted prices?

**Comment by jrbell19** [comment]
Yes, we are planning to adjust the data for dividends, however, we are still working to find a new quality vendor for the current dividend data itself. 

I do not have any kind of timeframe to expect quite yet, but we are in talks with vendors to source this data, as well as pulling it out of the SEC / PRs ourselves. Apologies for the inconvenience. I wish I had a better answer for you at this time.

**Comment by fingoldo** [comment]
Also really wishing you would have this capability of dividends adjustment.
